packages/gatsby-transformer-mdx-introspection/README.md

```diff
@@ -1,37 +1,42 @@
 # gatsby-transformer-mdx-introspection
 
-This plugin statically analyzes the nodes generated by `gatsby-plugin-mdx`
-and generates GraphQL Nodes that allow to find out what MDX-sourced pages
-contain which manually placed React components.
+This plugin statically analyzes the nodes generated by `gatsby-plugin-mdx` and generates GraphQL Nodes that allow users to find out what MDX-sourced pages contain which manually placed React components or automatically placed HTML elements.
```

Comment:
```suggestion
This plugin statically analyzes the nodes generated by `gatsby-plugin-mdx` and generates GraphQL nodes that allow users to find out what MDX-sourced pages contain which manually placed React components or automatically placed HTML elements.
```

Plus: what's an "automatically placed HTML element"?  If you mean the HTML generated from markdown syntax we could word that more explicitly.

Reply:
You're correct; I'll make that more clear in the README.